js/models/profile/Profile.js

```diff
@@ -13,7 +13,7 @@ export default class extends BaseModel {
       handle: '',
       location: '',
       moderator: false,
-      name: `ob ${Math.random().toString(36).slice(2)}`,
+      name: `OB ${Math.random().toString(36).slice(1).substring(1,7)}`,
```

Comment:
"OB 0" is a valid result, so that's working as intended. I suppose I should have said "should usually give you 6 characters, but has a small chance of being less."

If we want to guarantee 6 characters, we can do this:

```
`OB ${(Math.floor(Math.random() * 2116316159) + 60466176).toString(36)}`
```

Floor here is nicer than cutting it at the period, and this should generate a string between zzzzzz (the highest 6 character number in base 36) and 100000 (the lowest).

That's just off the top of my head, so suggestions on improving it are welcome.